webservices/docs.py

```diff
@@ -17,14 +17,11 @@
 item details with the other endpoints. If you are interested in individual donors, check
 out contributor information in schedule_a.
 
-If you would like to use the FEC's API programatically, you can sign up for
-your own API key below. But, you can still try out our API without
-an API key by using the web interface below and making use of a DEMO_KEY.
-Signing up for an API key will enable you to place up to 1,000 calls an hour.
-Each call is limited to 100 results per page. You can email questions, comments or
-a request to get a key for 120 calls per minute to [APIinfo@fec.gov](mailto:apiinfo@fec.gov). You can also
-ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data).
-API changes will also be added to this group in advance of the change.
+<b class="body" id="getting_started_head">Getting started with the openFEC API</b><br>
+If you would like to use the FEC's API programmatically, you can sign up for your own API key using our form. Alternatively, you can still try out our API without an API key by using the web interface and using DEMO_KEY. Note that when you use the openFEC API you are subject to the [Terms of Service](https://github.com/fecgov/FEC/blob/master/TERMS-OF-SERVICE.md) and [Acceptable Use policy](https://github.com/fecgov/FEC/blob/master/ACCEPTABLE-USE-POLICY.md).
+
+
+Signing up for an API key will enable you to place up to 1,000 calls an hour. Each call is limited to 100 results per page. You can email questions, comments or a request to get a key for 7,600 calls per hour (120 calls per minute) to <a href="mailto:APIinfo@fec.gov">APIinfo@fec.gov</a>. You can also ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data). API changes will also be added to this group in advance of the change.
```

Comment:
I’d like to edit this language, `You can also ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data). API changes will also be added to this group in advance of the change.`

I’d like to replace the first sentence with something like this, `You can also ask questions and discuss the data in a community led  [group](https://groups.google.com/forum/#!forum/fec-data).`

I’d like delete the second sentence, ~~API changes will also be added to this group in advance of the change.~~

Reply:
We can still post API changes in the Google group, but I dint want us to be obligated to do it.

Reply:
updated